ci/gpu/build.sh

```diff
@@ -5,4 +5,75 @@
 ##########################################
 set -e
 
-exit 1
\ No newline at end of file
+# Logger function for build status output
+function logger() {
+  echo -e "\n>>>> $@\n"
+}
+
+# Set path and build parallel level
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
+export PARALLEL_LEVEL=4
+
+# Set home to the job's workspace
+export HOME=$WORKSPACE
+
+################################################################################
+# SETUP - Check environment
+################################################################################
+
+logger "Check environment..."
+env
+
+logger "Check GPU usage..."
+nvidia-smi
+
+logger "Activate conda env..."
+source activate gdf
+
+export CUDA_VERSION_SHORT=${CUDA_VERSION%.*}
+conda install -c nvidia -c rapidsai -c rapidsai-nightly/label/cuda$CUDA_VERSION_SHORT -c numba -c conda-forge -c defaults cudf=0.6 nvgraph
```

Comment:
Can you define the cuDF version at the top in big letters so it's easy to find and update.